src/main/java/com/atlauncher/gui/tabs/Tab.java

```diff
@@ -17,8 +17,21 @@
  */
 package com.atlauncher.gui.tabs;
 
+/**
+ * Defines a tab that will display at the end of the UI
+ */
 public interface Tab {
+
+    /**
+     * Title to display to the user.
+     * If null is returned, the tab will not be displayed.
```

Comment:
I don't believe this statement is correct. The tab is still there, just without title. I wouldn't mention it honestly